templates/angular/igx-ts/projects/empty/files/angular.json

```diff
@@ -27,7 +27,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+			  "src/styles.scss"$(DefaultTheme)
```

Comment:
Indent formatting is off. Would prefer if it remains consistent.

Reply:
Done